packages/template-chakra-storefront/config/default.json

```diff
@@ -1,217 +0,0 @@
-{
```

Comment:
Bonus: removed the duplicate config file. We've moved to default.js, so this json file is no longer needed.

Reply:
Great to see this gone... can you please find and close the associated ticket @alexvuong mentioned that was open from the prior V4 iteration to move back to `default.js` instead of `default.json`?